tarea 2/Mensaje.html

```diff
@@ -0,0 +1,31 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Mensaje</title>
+</head>
+<body>
+    <h1>Mensajes</h1>
+    
+    <script>
+        alert("Hola soy mensaje");
+        alert(1);
+        alert(2);
+        alert(3);
+        alert(4);
+        alert(5);
+        alert(6);
+        alert(7);
+        alert(8);
+        alert(9);
+        alert(10);
+        alert(11);
+        alert(12);
+        alert(13);
+        alert(14);
+        alert(15);
+        alert("Gracias 15 mensaje");
```

Comment:
Esta tarea tiene un error.

Hay que preguntar al usuario cuántas veces quiere el mensaje. Después, hay que hacer un `for` con el número de veces del mensaje.